content/blog/2022-06-07-using-gcp-remotes-in-dvc.md

```diff
@@ -0,0 +1,158 @@
+---
+title: Syncing Data to GCP Storage Buckets
+date: 2022-06-07
+description: >
+  We're going to set up an GCP storage bucket remote in a DVC project.
+descriptionLong: >
+  Setting up a remote to make data versioning easier with DVC is a common need
+  so we're going to go through a tutorial for doing this with GCP.
+picture: 2022-06-07/gcp-in-dvc.png
+pictureComment: Using GCP Remotes in DVC
+author: milecia_mcgregor
+commentsUrl: https://discuss.dvc.org/t/preventing-stale-models-in-production/1137
+tags:
+  - MLOps
+  - DVC
+  - Git
+  - DVC Remotes
+  - Collaboration
+---
+
+When you’re working on a data science project that has huge datasets, it’s
+common to store them in cloud storage. You’ll also be working with different
+versions of the same datasets to train a model, so it’s crucial to have a tool
+that enables you to do this quickly and easily. That’s why we’re going to do a
+quick walkthrough of how to set up a remote in a GCP storage bucket and handle
+data versioning with [DVC](https://dvc.org/doc).
+
+We’ll start by creating a new storage bucket in our GCP account, then we’ll show
+how you can add DVC to your project, and finally, we’ll make updates to the
+dataset with DVC commands. We’ll be working with
+[this repo](https://github.com/iterative/stale-model-example) if you want an
```

Comment:
Is there a way to get the data for the repo? Following along, it's hard to figure out what to do because I can't pull the original data.

Reply:
Done! I pushed the .csv files to that stale model repo.